### PR TITLE
SAK-46809 Assignments Add allow extension to Sakai Grader

### PR DESCRIPTION
**Pull request by davidpbauer**
This makes the existing allow extension feature in Assignemnts
available in Sakai Grader in the same way it exists in the classic
grader view.

### CODE REVIEW

File: webcomponents/tool/src/main/frontend/js/grader/submission.js [hunk 1]
```diff
@@ -90,6 +94,13 @@ export class Submission {
...
   }
+  
+  set extensionAllowed(value){
+    this._extensionAllowed = value;
+    this.extensionDate = moment(this.assignmentCloseTime).valueOf();
+  }
 
   get resubmitsAllowed() { return this._resubmitsAllowed; }
+  
+  get extensionAllowed(){return this._extensionAllowed;}
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/grader/submission.js:105]:
```suggestion
  get extensionAllowed() { return this._extensionAllowed; } 
```

**Comment by davidpbauer** [comment]
Thanks for the changes!

**Review by jesusmmp — APPROVED** [review]
LGTM!

**Review by adrianfish — APPROVED** [review]
Thanks for the changes.

**Pull request merged**